addons/packages/secretgen-controller/0.7.1/package.yaml

```diff
@@ -3,23 +3,41 @@ kind: Package
 metadata:
   name: secretgen-controller.community.tanzu.vmware.com.0.7.1
 spec:
+  valuesSchema:
+    openAPIv3:
+      type: object
+      additionalProperties: false
```

Comment:
Could we add `description` here? See example https://github.com/kubernetes-sigs/cluster-api/blob/main/config/crd/bases/cluster.x-k8s.io_clusters.yaml#L30

Reply:
yes that can be added